spec/spec_helper.rb

```diff
@@ -4,5 +4,5 @@
 CodeClimate::TestReporter.start
 
 require 'ostruct'
-require 'pry-byebug'
+require 'pry'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.